modules/appeals_api/app/middleware/sidekiq_job_duration_middleware.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+require 'datadog/statsd'
+
+class SidekiqJobDurationMiddleware
+  # Job duration will only be logged for workers from these modules
+  MODULE_NAMES = %w[AppealsApi VAForms VBADocuments].freeze
```

Comment:
While I like the overall idea here, I think a better approach would be to have each sidekiq job able to opt-in to this duration logging, similar to how we did for [Sidekiq::MonitoredWorker](https://github.com/department-of-veterans-affairs/vets-api/pull/599)

This way each team could decide which jobs needed duration logging rather than capturing all jobs in a module, and conflicts could be avoided from different teams trying to modify the array.

Reply:
I added a new module, `Sidekiq::BenchmarkLoggingWorker` to mark the jobs that should be benchmarked  - for now, I've only applied it to a few jobs (specifically the flipper status alert jobs in each of the three modules we work with), since I was a bit hesitant to apply it everywhere right away